server/sonar-web/src/main/js/apps/projectActivity/components/forms/AddGraphMetric.js

```diff
@@ -20,20 +20,24 @@
 // @flow
 import React from 'react';
 import classNames from 'classnames';
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Remove this unused import of 'classNames'. ([javascript:S1128](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=javascript%3AS1128))

[See it in SonarQube](https://next.sonarqube.com/sonarqube/project/issues?id=org.sonarsource.sonarqube%3Asonarqube&pullRequest=3149&issues=AWIgFTRboomSmF0fSGEm&open=AWIgFTRboomSmF0fSGEm)

[ISSUE_KEY]: AWIgFTRboomSmF0fSGEm